src/services/feed_manager.py

```
@@ -65,53 +67,55 @@
                     invalid_feed = Feed(
                         title=feed.title,
                         url=url,
                         genre=feed.genre,  # Preserve original genre
-                        description=f"{feed.description}\nValidation Error: {error}"
+                        description=f"{feed.description}\nValidation Error: {error}",
                     )
                     invalid_feeds_by_genre[feed.genre].append(invalid_feed)
                 await self._save_invalid_feeds(invalid_feeds_by_genre)
-            
+
             logging.info(f"Loaded {len(self.feeds)} valid feeds from OPML file")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)